app/Moderation/ModerationServiceProvider.php

```diff
@@ -28,6 +28,7 @@ public function register()
                 $app->make('MyBB\Core\Moderation\Moderations\DeletePost'),
                 $app->make('MyBB\Core\Moderation\Moderations\DeleteTopic'),
                 $app->make('MyBB\Core\Moderation\Moderations\Close'),
+                $app->make('MyBB\Core\Moderation\Moderations\Stick'),
```

Comment:
Hi, to keep with the code style, can you update this to `\MyBB\Core\Moderation\Moderations\Stick::class`?